<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ID_input">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>enter your ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>enter your name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="department_input">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>enter your department</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cgpa_input">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>enter your CGPA (0.00)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="semester_selection">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>select semester</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="insert_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INSERT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayall_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DISPLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="admin_exit_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: 4px solid rgb(85, 170, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Department</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Semester</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CGPA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>ID_input</zorder>
   <zorder>name_input</zorder>
   <zorder>department_input</zorder>
   <zorder>cgpa_input</zorder>
   <zorder>semester_selection</zorder>
   <zorder>insert_button</zorder>
   <zorder>delete_button</zorder>
   <zorder>search_button</zorder>
   <zorder>displayall_button</zorder>
   <zorder>admin_exit_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ID_input</sender>
   <signal>editingFinished()</signal>
   <receiver>ID_input</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>name_input</sender>
   <signal>editingFinished()</signal>
   <receiver>name_input</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>department_input</sender>
   <signal>editingFinished()</signal>
   <receiver>department_input</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>semester_selection</sender>
   <signal>activated(int)</signal>
   <receiver>semester_selection</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cgpa_input</sender>
   <signal>editingFinished()</signal>
   <receiver>cgpa_input</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>insert_button</sender>
   <signal>clicked()</signal>
   <receiver>insert_button</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_button</sender>
   <signal>clicked()</signal>
   <receiver>delete_button</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_button</sender>
   <signal>clicked()</signal>
   <receiver>search_button</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayall_button</sender>
   <signal>clicked()</signal>
   <receiver>displayall_button</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>admin_exit_button</sender>
   <signal>clicked()</signal>
   <receiver>admin_exit_button</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
